src/runtime/runtime.js

```diff
@@ -580,7 +580,6 @@ export class ConfiguredRuntime {
     preconnect.prefetch('$assets$/loader.svg');
     LinkCompleteFlow.configurePending(this);
     PayCompleteFlow.configurePending(this);
-    this.payClient_.preconnect(preconnect);
```

Comment:
I think preconnecting is safe but wary of putting any prefetch code since this issue was about cookies being dropped by pay. Adding in preconnecting within Runtime directly.